docs/index.rst

```diff
@@ -1,110 +1,87 @@
-   environments
- - watching specific files or directories for changes in environment
-   filename specifications
- - provides a REST api for managing environments (which a jupyterlab
-   plugin is being actively developed for)
- - provides a command line utility for interacting with conda-store
-   conda-store env [create, list]
- - full role based access controls (rbac) around environment view,
-   creation, update, and deletion.
-
-It **builds** conda specifications in a scalable manner using N
-workers communicating using Celery to keep track of queued up
-environment builds.
-
-It **serves** conda environments via a filesystem, lockfiles,
-tarballs, and soon a docker registry. Tarballs and docker images can
-carry a lot of bandwidth which is why conda-store integrates
-optionally with s3 to actually serve the blobs.
-
-Contents
-========
-
 .. toctree::
    :maxdepth: 2
+   :hidden:
 
    installation
    user_guide
    administration
    contributing
    api
 
-Indices and tables
-==================
+.. image:: _static/images/conda-store-logo-vertical-lockup.png
+    :alt: conda-store logo
+    :scale: 50%
+    :align: center
+    :class: dark-light p-2
+
+
+.. rst-class:: center
+
+**Data science environments, for collaboration.**
+
+----
+
+.. rst-class:: center
+
+conda-store is an open source tool for managing data science environments in collaborative teams.
+It provides flexible, yet reproducible, environments while enforcing best practices throughout your environment's life cycle.
+
+.. panels::
+    :container: container-lg pb-3
+    :column: col-lg-4 col-md-4 col-sm-6 col-xs-12 p-2
+
+    **🧶  Flexible**
+    ^^^^^^^^^^^^^^
+
+    .. rst-class:: left
+
+    Create and update environments quickly using a graphical UI or a YAML editor.
+
+    .. rst-class:: left
+
+    All environments are automatically version-controlled, and available for use.
+
+    ---
+    **💫  Reproducible**
+    ^^^^^^^^^^^^^^^^^^
+
+    .. rst-class:: left
+
+    Share environments effortlessly through the auto-generated artifacts: lockfile, docker image, YAML file, or tarball.
+
+    .. rst-class:: left
+
+    Exact versions of all packages and their dependencies are pinned in these artifacts.
+
+    ---
+    **⚖️  Goveranance**
+    ^^^^^^^^^^^^^
+
+    .. rst-class:: left
+
+    Access admin-approved packages and channels, and request new ones when needed.
+
+    .. rst-class:: left
+
+    Admins have role-based access management, to allow users to share environments across (and only with) their team.
+
+.. rst-class:: center
+
+The conda packaging system works well for individual users, but it can get tricky to reproduce environments reliably to share with colleagues. On the other hand, Docker containers can ensure reproducibility, but aren't friendly to individual users who may need new packages or versions for daily work. conda-store resolve this friction, and provides a balance of both the familiarity of conda and the robustness of containers.
```

Comment:
```suggestion
The conda packaging system works well for individual users, but it can get tricky to reproduce environments reliably to share with colleagues. On the other hand, Docker containers can ensure reproducibility, but aren't friendly to individual users who may need new packages or versions for daily work. conda-store resolves this friction, and provides a balance of both the familiarity of conda and the robustness of containers.
```